source/documentation/self-hosted/chap-Deploying_Self-Hosted_Engine.html.md

```diff
@@ -314,13 +314,13 @@ You can deploy a self-hosted engine from the command line using `hosted-engine -
 
         * Configure the volume as follows:
 
-            gluster volume set _volume_ cluster.quorum-type auto
-            gluster volume set _volume_ network.ping-timeout 10
-            gluster volume set _volume_ auth.allow \*
-            gluster volume set _volume_ group virt
-            gluster volume set _volume_ storage.owner-uid 36
-            gluster volume set _volume_ storage.owner-gid 36
-            gluster volume set _volume_ server.allow-insecure on
```

Comment:
Why server.allow-insecure  and cluster.quorum-type auto were removed?

Reply:
cluster.quorum-type=auto is part of group virt. and server.allow-insecure was a workaround to an earlier issue connecting to the gluster servers which is no longer needed